modules/KalturaSupport/components/chapters/chapters.js

```diff
@@ -981,8 +991,13 @@
 				if (!this.getPlayer().canAutoPlay() && this.getPlayer().firstPlay){
 					this.getPlayer().sendNotification('doPlay');
 				}
-				// see to start time and play ( +.1 to avoid highlight of prev chapter )
-				this.getPlayer().sendNotification('doSeek', ( this.mediaList[mediaIndex].startTime ) + 0.1);
+				if (this.embedPlayer.isDVR()) {
+					var seekTo = this.mediaList[mediaIndex].startTime - this.embedPlayer.evaluate('{mediaProxy.entry.firstBroadcast}') + this.embedPlayer.kalturaPlayerMetaData.firstBroadcast - this.embedPlayer.kalturaPlayerMetaData.currentBroadcastStartTime;
```

Comment:
1.mediaProxy.entry.firstBroadcast. currentBroadcastStartTime is  not something you should use...
- You don't know which DC is serving you, the currentBroadcastStartTime is not per DC, and the two DC are not in sync..
- It's "deprecatd", and we plan to remove this one day, it's not a value you should depend 
2. the DVR window can be smaller than the session so even if we update currectly the currentBroadcastStartTime you will have a problem here